### PR TITLE
VZ-7127 Secondary cleanup PR

### PR DESCRIPTION
**Pull request by bryantidd**
Cleanup code and add clarity to concerns

### CODE REVIEW

File: platform-operator/main.go [hunk 1]
```diff
@@ -135,105 +135,111 @@ func main() {
 	// initWebhooks flag is set when called from an initContainer.  This allows the certs to be setup for the
```
Comment (jmaron99) [inline platform-operator/main.go:135]: comment appears to be out of date

File: platform-operator/main.go [hunk 2]
```diff
@@ -135,105 +135,111 @@ func main() {
...
 	// validatingWebhookConfiguration resource before the operator container runs.
 	if config.InitWebhooks {
```
Comment (jmaron99) [inline platform-operator/main.go:137]: also looks like config.WebhooksEnabled is no longer used?

File: platform-operator/main.go [hunk 3]
```diff
@@ -135,105 +135,110 @@ func main() {
...
-		log.Debug("Creating certificates used by webhooks")
-		caCert, err := certificate.CreateWebhookCertificates(config.CertDir)
```
Comment (jmaron99) [inline platform-operator/main.go:139]: I'll need to modify this method to allow for the update of my webhook with the cert config.  I'll add "webhookName" as an argument.

**Review by jmaron99 — APPROVED** [review]

File: platform-operator/main.go [hunk 4]
```diff
@@ -135,105 +135,110 @@ func main() {
...
-			os.Exit(1)
-		}
+		startWebhookServers(config, log)
+	} else {
+		reconcilePlatformOperator(config, log)
+	}
+}
+
+func startWebhookServers(config internalconfig.OperatorConfig, log *zap.SugaredLogger) {
+	// initWebhooks flag is set when called from an webhook deployment.  This allows separation of webhooks and operator
```
Comment (bobdonat) [inline platform-operator/main.go:145]: comment looks to out of place

Comment (bryantidd) [reply platform-operator/main.go:145]: I'll move it up, I fixed the comment content, but didn't move it to the right place now

File: platform-operator/main.go (hunk 2 above)
Comment (bobdonat) [reply platform-operator/main.go:136]: I noticed the same thing.  Prior to this we allowed the webhooks to be initialized but not be enabled.  Not sure why that was needed. If we no longer want that then the flag should be removed.

File: platform-operator/main.go [hunk 5]
```diff
@@ -132,108 +132,111 @@ func main() {
...
-			os.Exit(1)
-		}
+		startWebhookServers(config, log)
+	} else {
+		reconcilePlatformOperator(config, log)
+	}
+}
+
+func startWebhookServers(config internalconfig.OperatorConfig, log *zap.SugaredLogger) {
+
```
Comment (pfmackin) [inline platform-operator/main.go:144]: Can you remove the blank line

File: platform-operator/main.go [hunk 6]
```diff
@@ -132,108 +132,111 @@ func main() {
...
-		if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-			log.Errorf("Failed starting controller-runtime manager: %v", err)
-			os.Exit(1)
-		}
+		startWebhookServers(config, log)
+	} else {
+		reconcilePlatformOperator(config, log)
+	}
+}
+
```
Comment (pfmackin) [inline platform-operator/main.go:142]: Can you add a function description explaining that this code only runs in the VPO-Webhook, not the VPO.  Might be confusing to someone looking at this.

File: platform-operator/main.go (hunk 2 above)
Comment (bryantidd) [reply platform-operator/main.go:136]: Will do

File: platform-operator/main.go (hunk 5 above)
Comment (bryantidd) [reply platform-operator/main.go]: yes

File: platform-operator/main.go (hunk 6 above)
Comment (bryantidd) [reply platform-operator/main.go:142]: Yes

**Review by pfmackin — APPROVED** [review]

**Pull request merged**